src/angularjs/src/app/places/detail/places-detail.html

```diff
@@ -32,20 +32,25 @@ <h4 class="sidebar-title">{{placeDetail.place.state_abbrev}}</h4>
                             <div class="tooltip" data-title="On average, this is how well the bike network gets people to the places they want to go."><i class="icon-info-circled"></i></div>
                         </label>
                     </div>
+                    <div class="column">
+                        <button class="btn btn-default" type="button"
+                            ui-sref="places.compare({place1: placeDetail.place.uuid})">Compare
+                        </button>
+                    </div>
                     <div class="column text-right">
-
-                    <div class="dropdown" uib-dropdown is-open="status.isopen"
-                        ng-if="placeDetail.downloads">
-                      <button id="single-button" type="button" class="btn btn-default btn-s" uib-dropdown-toggle ng-disabled="disabled" id="downloadButton">
-                        Download <span class="caret"></span>
-                      </button>
-                      <ul class="dropdown-menu" uib-dropdown-menu role="menu"
-                        aria-labelledby="downloadButton">
-                        <li ng-repeat="item in placeDetail.downloads">
-                            <a role="menuitem" href="{{item.url}}"
-                                class="compare-remove">{{item.label}}</a>
-                        </li>
-                      </ul>
+                        <div class="dropdown" uib-dropdown is-open="status.isopen"
```

Comment:
The compare button should sit inside of the same `.column` where the download button is located. The middle `.column` should be deleted.

Within the scss file `_location.scss` starting on line 2 add the styling:
```
.dropdown {
  display: inline;
}
```

As a minor styling detail I would recommend the following change in the scss file `_button.scss`, line 74: 
`color: $shade-dark;` to `color: #000;`

After these changes everything should be good to go.